nodes/analyzer/evaluate_image.py

```diff
@@ -183,10 +186,16 @@ def process(self):
                     red[0].append(imag[index])
                     green[0].append(imag[index+1])
                     blue[0].append(imag[index+2])
+                    alpha[0].append(imag[index+3])
                 else:
                     red[0].append(0)
                     green[0].append(0)
                     blue[0].append(0)
+                    alpha[0].append(0)
+        if outputs['BW'].is_linked:
+            bw[0] = [(r+g+b)/3 for r,g,b in zip(red[0], green[0], blue[0])]
```

Comment:
`(r+g+b)/3`  ?
not to be picky but...
![image](https://user-images.githubusercontent.com/619340/48660627-1afa2e00-ea65-11e8-857d-21cf5c6d2c54.png)
https://en.wikipedia.org/wiki/Grayscale

Reply:
You are not picky, it's my bad. I was to hurried :P, I'll fix that ;)

Reply:
: ]